saleor/webhook/payloads.py

```diff
@@ -151,3 +155,37 @@ def generate_product_payload(product: "Product"):
         },
     )
     return product_payload
+
+
+def generate_sample_payload(event_name):
+    if event_name == WebhookEventType.CUSTOMER_CREATED:
+        users = User.objects.filter(is_staff=False, is_active=True)[:2]
```

Comment:
Is there a reason to always return `2` objects from the queryset in this function? Since this is a list you could always return one object and pack it into a list, unless there is some requirement from Zapier.

Reply:
It can be a sinlge object in list. I will change it